packages/create-sitecore-jss/src/templates/angular/package.json

```
@@ -19,7 +19,7 @@
     "e2e": "ng e2e",
     "jss": "jss",
     "start:connected": "npm-run-all --serial bootstrap start:angular start:watch-components",
-    "build": "npm-run-all --serial bootstrap --serial build:client build:server",
+    "build": "cross-env-shell JSS_MODE=production \"npm-run-all --serial bootstrap --serial build:client build:server\"",
```

Like this before, but seems like there's an unnecessary `--serial` here?
```suggestion
    "build": "cross-env-shell JSS_MODE=production \"npm-run-all --serial bootstrap build:client build:server\"",
```